modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/OperationExecutionTestCase.java

```diff
@@ -268,6 +268,7 @@ public void operationWithStaticInlinePojoParameter() throws Exception {
   }
 
   @Test
+  // This cannot be validated at the AST because the exception results of a provided expression evaluating to null
```

Comment:
use appropriate comment format for a method or add this this comment in the method.